### PR TITLE
ROX-13489: add automated tests for Clair v4 integration

### PR DESCRIPTION
**Pull request by RTann**
## Description

Add Clair v4 k8s deployment and ImageScanningTest to ensure Clair v4 scanning integration works properly

## Checklist
- [x] Investigated and inspected CI test results
- [x] Unit test and regression tests added
- ~[ ] Evaluated and added CHANGELOG entry if required~
- ~[ ] Determined and documented upgrade steps~
- ~[ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))~

## Testing Performed

CI

### CODE REVIEW

File: pkg/features/list.go [hunk 1]
```diff
@@ -54,7 +54,7 @@ var (
 	ProcessesListeningOnPort = registerFeature("Enable Processes Listening on Port", "ROX_PROCESSES_LISTENING_ON_PORT", false)
 
 	// ClairV4Scanner enables Clair v4 as an Image Integration option
-	ClairV4Scanner = registerFeature("Enable Clair v4 as an Image Integration option", "ROX_CLAIR_V4_SCANNING", false)
+	ClairV4Scanner = registerFeature("Enable Clair v4 as an Image Integration option", "ROX_CLAIR_V4_SCANNING", true)
```
Comment (RTann) [inline pkg/features/list.go:57]: is there a way to feature flag the qa groovy test?

Comment (gavin-stackrox) [reply pkg/features/list.go:57]: I think https://github.com/stackrox/stackrox/blob/f8bdc4ea695505af176c85cd15de7699f0f677e0/tests/e2e/lib.sh#L85 if you want it for all e2e.

File: qa-tests-backend/scripts/run-part-1.sh [hunk 2]
```diff
@@ -74,6 +74,8 @@ test_part_1() {
 
     rm -f FAIL
 
+    "$ROOT/scripts/ci/clairv4/deploy.sh" qa-clairv4
```
Comment (gavin-stackrox) [inline qa-tests-backend/scripts/run-part-1.sh:77]: I think this makes more sense at the end of `config_part_1()`

File: qa-tests-backend/scripts/run-part-1.sh [hunk 3]
```diff
@@ -97,6 +99,8 @@ test_part_1() {
         make -C qa-tests-backend bat-test || touch FAIL
     fi
 
+    "$ROOT/scripts/ci/clairv4/teardown.sh" qa-clairv4
```
Comment (gavin-stackrox) [inline qa-tests-backend/scripts/run-part-1.sh:102]: I think I'd skip this. Teardown is not typically done.

**Review by gavin-stackrox — CHANGES_REQUESTED** [review]

**Review by gavin-stackrox — APPROVED** [review]
v4 :vroom:

**Pull request merged**